lib/df-detection.c

```diff
@@ -64,17 +61,27 @@ void end_system_call(void)
 		current->addresses = NULL;
 	}
 }
-void print_pairs(void)
+void report(void)
 {
 	int i;
-	for (i = 0; i < current->df_index; i++) {
-		pr_err("First %px len %lu \nSecond %px len %lu\n",
-		       current->pairs[i].first->start_address,
-		       current->pairs[i].first->len,
-		       current->pairs[i].second->start_address,
-		       current->pairs[i].second->len);
+	pr_err("BUG: Intersection Detected \n ");
+	pr_err("==================================================================\n");
+	if (panic_on_warn) {
+		pr_err("System Call Number: %ld\n", current->syscall_num);
```

Comment:
I still think printing the symbol from sys_call_table[] is worthwhile here.

Reply:
yeah i know i will use sys_call_table instead of just printing systemcall number, but i just push the updates after making sure that this report function is printing in syzkaller crash report. to make everything up-to-date.